src/content/developers/tutorials/integrating-your-smart-contract-with-the-frontend/index.md

```diff
@@ -0,0 +1,920 @@
+* Sign Ethereum transactions using Metamask
+
+For this dApp, we'll be using [React](https://reactjs.org/) as our frontend framework; however, it's important to note that we won't be spending much time breaking down its fundamentals, as we'll mostly be focusing on bringing Web3 functionality to our project.
+
+As a prerequisite, you should have a beginner-level understanding of React—know how components, props, useState/useEffect, and basic function calling works. If you've never heard of any of those terms before, we recommend that you check out this [Intro to React tutorial](https://reactjs.org/tutorial/tutorial.html). For the more visual learners, we highly recommend this awesome Net Ninja [Full Modern React Tutorial](https://www.youtube.com/playlist?list=PL4cUxeGkcC9gZD-Tvwfod2gaISzfRiP9d) video series.
+
+## Step 1: Clone the starter files {#step-1-starter-files}
+
+First, go to the [hello-world-part-four github repository](https://github.com/alchemyplatform/hello-world-part-four-tutorial) to get the starter files for this project. Clone this repository into your local environment.
+
+### Don't know how to clone a repository? Check out [this guide](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) from Github.
+
+When you open this cloned `hello-world-part-four` repository, you'll notice that it contains two folders: `starter-files` and `completed`.
+
+* `starter-files` contains the starter files \(essentially the React UI\) for this project. In this tutorial, **we will be working in this directory**, as you learn how to bring this UI to life by connecting it to your Ethereum wallet and the Hello World smart contract that you published on Etherscan in [Part 3](https://docs.alchemy.com/alchemy/tutorials/hello-world-smart-contract/submitting-your-smart-contract-to-etherscan).
+* `completed` contains the entire completed tutorial and is there for you as a **reference** **if you get stuck.**
+
+Next, open your copy of `starter-files` to your favorite code editor \(at Alchemy, we we're big fans of [VSCode](https://code.visualstudio.com/download)\), and then navigate into your `src` folder.
+
+All of the code we'll write will live under the `src` folder. We'll be editing the `HelloWorld.js` component and the `util/interact.js` javascript files to give our project Web3 functionality.
+
+## Step 2: Check out the starter files {#step-2-check-out-the-starter-files}
+
+Before we start coding, it's super important that we figure out what's already provided for us in the starter files.
+
+### Get your react project running
+
+Let's start by running the React project in our browser. The beauty of React is that once we have our project running in our browser, any changes we save will be updated live in our browser.
+
+To get the project running, navigate to the root directory of the `starter-files` folder, and the run `npm install` in your terminal to install the dependencies of the project:
```

Comment:
Technically they don't need to be at the root as long as no nested folder has a package.json file just FYI